src/components/application_manager/src/mobile_message_handler.cc

```diff
@@ -104,7 +104,7 @@ application_manager::Message* MobileMessageHandler::HandleIncomingMessageProtoco
   }
   LOG4CXX_DEBUG(logger_, "Incoming RPC_INFO: " <<
                          (out_message->connection_key() >> 16) <<", "<<
-                         GetMessageTypes[out_message->type()] <<", "<<
+                         //GetMessageTypes[out_message->type()] <<", "<<
```

Comment:
We the special practice for commented code: as all other hot-workaround it need to be commented with ticket id where is will be fixed.
 For example
```// TODO(ANosach): APPLINK-99999999 - Implement streaming for MessageType```
So, Tema, please, be kind to follow this rule. Thanks!